Services/CO.CDP.OrganisationInformation.Persistence/DatabaseFormRepository.cs

```diff
@@ -33,6 +40,16 @@ public async Task SaveFormAsync(Form form)
 
     #endregion
 
+    #region Shared Consents Methods
+
+    public async Task<SharedConsent?> GetSharedConsentAsync(Guid formId, Guid organisationId)
+    {
+        return await context.Set<SharedConsent>()
+            .FirstOrDefaultAsync(s => s.Form.Guid == formId && s.Organisation.Guid == organisationId);
```

Comment:
Actually, we should find the consent that has no shared code assigned. There should only ever be one.

Reply:
done.